Assets/Splash/SplashSequence.cs

```diff
@@ -0,0 +1,37 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.SceneManagement;
+
+public class SplashSequence : MonoBehaviour
+{
+    public static int SceneNumber;
```

Comment:
Question out of curiosity: What's the reason to keep this value as a state of class instance?